src/raygun.network-tracking.js

```diff
@@ -157,6 +157,12 @@ window.raygunNetworkTrackingFactory = function(window, Raygun) {
         var method = (options && options.method) || 'GET';
         var initTime = new Date().getTime();
 
+
+
+
+        // One idea is to have a typeof bonusUrl 
+        //Where we extract a real url and a reporting one
+        //The user would then need to call fetch({realUrl: "http//:abv.com/api/allTheThings", reportingUrl: "http//:abv.com/api/allTheThings#GetUser(zyz)"}, BODY)
```

Comment:
I can see something like this working, although, I feel this has definitely fallen into the realm of a feature request.
This is a good thing because with this it can get a group discovery, project plan & appropriate testing if selected for development rather than being an item on someone's list.

I've validated that there is simply no payload information available inside the onBeforeXHR or onBeforeRUM calls so I think I'll put a cork in it and move on. I dabbled with the idea of monkey-patching the window fetch/XHR prototypes but this is more of a hack & I don't think it would ever be something we recommend to customers.